lcls_naming_tool/lcls_naming_tool.py

```diff
@@ -261,30 +252,35 @@ def validate(user_input):
 
 def main():
 
-    display_version()
+    version = get_version()
```

Comment:
we can move this into the `if args.version:` block to avoid making this call and not using it